nitrocli/src/commands.rs

```diff
@@ -0,0 +1,149 @@
+// commands.rs
```

Comment:
It would be great to make the introduction of this file a separate commit. I've actually done that already in [wip/argparse](https://github.com/d-e-s-o/nitrocli/commits/wip/argparse) in order to make reviewing easier. Is that okay with you?